app/models/financial_aid/financial_resources.rb

```diff
@@ -0,0 +1,38 @@
+module FinancialAid
+  class FinancialResources
+    include Cache::CachedFeed
+    include LinkFetcher
+
+    def get_feed
+      get_financial_resources
+    end
+
+    def get_financial_resources
+      {
+        links: financial_resources_links
+      }
+    end
+
+    def financial_resources
+      @financial_resources ||= EdoOracle::FinancialAid::Queries.get_financial_resources_links()
+    end
+
+    def financial_resources_links
+
+      links = {}
+
+      financial_resources.each do |cs_link|
+        links[cs_link['url_id']] = fetch_link(cs_link['url_id'], link_params)
+      end
+
+      links
+    end
```

Comment:
This can be greatly simplified. Also, removing the empty whitespace.

```
Hash.new.tap do |links|
  financial_resources.each do |cs_link|
    links[cs_link['url_id']] = fetch_link(cs_link['url_id'], link_params)
  end
end
```

Tap `Yields self to the block, then returns self.` That is, allows you to instantiate the variable show that it's the thing being returned.

Reply:
Thanks!  TIL...